assays/atac.md

```diff
@@ -0,0 +1,54 @@
+# ATACseq
+
+TODO
```

Comment:
@scvannost could you write up a fitting description for this?

Reply:
in my commit